content/client-server-api/modules/push.md

```diff
@@ -521,8 +521,43 @@ Definition:
 }
 ```
 
+**`.m.rule.is_user_mention`**
```

Comment:
consider giving these anchors so that they can be referenced from `mentions.md`:

```suggestion
<a id="_m_rule_is_user_mention"/> **`.m.rule.is_user_mention`**
```